Implemented for MC 1.15.x

SamCZ opened the pull request:
Forked https://github.com/SamCZ/zLib and fixed it for mc 1.15.x

If you find any bug, please write a comment.

Thanks.

AmauryCarrade commented:
I'll look into it as soon as possible.

You make zLib only support 1.15+; that's something I had in mind since a long time but I was always in doubt so I'll re-think about it as it solve a lot of bugs, and older versions can still be built against the old zLib versions…

SamCZ commented:
OK thanks, there was only one major issue because minecraft since 1.15.+ changed ntb tag functinality, but i managed to solved it by just enabling private constructor in your code, but there is another thing... the advacements, for basic functinality i just deleted all code that implements old achievements and i dont know if it is needed fo some functionality in your `ImageOnMap`.

I just needed the plugin working as fast as possible, because i was too tired at night and i wanted to show something to my friend with the tool in 1.15.1 :D

If you need i can help you with implementing the advancements or something.

Vlammar commented:
We have released a fix for 1.15.X.
